_posts/2024-01-15-the-gecoding-toolbox-is-being-deprecated.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'The Gecoding Toolbox Is Being Deprecated'
+author:
+  display_name: Scott Davis
+  email: stdavis@utah.gov
+date: 2024-01-15 16:30:36
+categories:
+  - Featured
+  - Developer
+tags:
+  - geocoding
+  - python
+---
+
+
+## First, Some History
+
+Back in 2014, we released a custom python toolbox for ArcGIS Desktop called the [Geocoding Toolbox](https://github.com/agrc/geocoding-toolbox). It allowed users to geocode Utah addresses using our [web API](https://api.mapserv.utah.gov/#geocoding). It was a great tool for the time, but there are now better options available. For this reason we have decided to deprecate the toolbox to allow us to focus our efforts and simplify our offerings.
+
+## What Are the Better Options?
+
+For those who are using Esri products including ArcGIS Pro and ArcGIS Online, our [Masquerade](https://github.com/agrc/masquerade) service will allow you to access our geocoding service without any special software. This service enables you to geocode Utah addresses and search for features in our [Open SGID](https://gis.utah.gov/data/) datasets. It can be used in the same way as any other locator. See the [project page](https://github.com/agrc/masquerade?tab=readme-ov-file#masquerade) for more details.
```

Comment:
```suggestion
For those who are using Esri products including ArcGIS Pro and ArcGIS Online, our [Masquerade](https://github.com/agrc/masquerade) service will allow you to access our geocoding service without any downloaded software. This service enables you to geocode Utah addresses and search for features in our [Open SGID]({% link /data/index.html %}) datasets. It can be used in the same way as any other locator. See the [project page](https://github.com/agrc/masquerade?tab=readme-ov-file#masquerade) for more details.
```